### PR TITLE
Cron jobs for  OTK DB maintenance schedule tasks.

### PR DESCRIPTION
**Pull request by APIM-KiranVaddadi**
**Description of the change**

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: charts/gateway/README.md [hunk 1]
```diff
@@ -499,9 +499,10 @@ Limitations:
 | `otk.job.image.repository`        | Image Repositor | `caapim/otk-install`
 | `otk.job.image.tag`               | Image Tag. (OTK version) | `4.6`
 | `otk.job.image.pullPolicy`        | Image Pull Policy | `IfNotPresent`
-| `otk.job.image.labels`            | Job lables | {}
-| `otk.job.image.nodeSelector`      | Job Node selector | {}
-| `otk.job.image.tolerations`       | Job tolerations | []
+| `otk.job.scheuledTasks`           | Valid only for ephemeral gateway. Creates cronJobs for each OTK DB maintenance schedule tasks |
```
Comment (ab-sub) [inline charts/gateway/README.md]: please correct the naming

Comment (ab-sub) [reply charts/gateway/README.md]: also in few other places

File: charts/gateway/templates/otk-scheduled-task-jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          - name: schedule-task
+            image: curlimages/curl:7.72.0
+            {{- if $.Values.containerSecurityContext }}
+            securityContext: {{- toYaml $.Values.containerSecurityContext | nindent 14 }}
+            {{- end }}
+            imagePullPolicy: IfNotPresent
+            command:
+            - /bin/sh
+            - -c
+            - "curl -v --request POST 'https://{{ template "gateway.fullname" $ }}:8443/oauth/dbmaintenance?task={{ .name }}' --insecure"
```
Comment (ab-sub) [inline charts/gateway/templates/otk-scheduled-task-jobs.yaml:56]: please update to dbMaintenance

File: charts/gateway/README.md (hunk 1 above)
Comment (APIM-KiranVaddadi) [reply charts/gateway/README.md]: updated

File: charts/gateway/templates/otk-scheduled-task-jobs.yaml (hunk 2 above)
Comment (APIM-KiranVaddadi) [reply charts/gateway/templates/otk-scheduled-task-jobs.yaml:56]: updated

**Review by ab-sub — APPROVED** [review]

**Pull request merged**